src/content/post/2024-08-15-kuzu-v-0.5.1.md

```diff
@@ -9,51 +9,62 @@ authors: ["team"]
 tags: ["cypher"]
 ---
 
-This release comes with a bunch of bug fixes and several new features and improvements.
+This release comes with several bug fixes and a much awaited feature: in-memory mode for Kùzu to quickly create temporary and in-memory databases!
+Please see the [release notes](https://github.com/kuzudb/kuzu/releases/tag/v0.5.1) for the bugfixes. In the rest of this post,
```

Comment:
@prrao87 can u also change the link here to `https://github.com/kuzudb/kuzu/releases/tag/v0.6.0`? I've switched the branch from 0.5.1 to 0.6.0 in main repo.

Reply:
Yes, will do.